ompy/extractor.py

```diff
@@ -58,6 +61,8 @@ class Extractor:
         seed (int): Random seed for reproducibility of results.
         resolution_Ex (float or np.ndarray, optional): Resolution (FWHM) along
             Ex axis (particle detector resolution). Defaults to 150 keV
+        rel_err_missing (float): Relative error used for points that cannot be
+            estimated by error_estimator object.
```

Comment:
when would that be the case?

Reply:
Sometimes if the statistics are low some of the realizations may have different number of points in the extracted NLD and GSF (values that are not nan).